scripts/prettier/index.js

```diff
@@ -38,14 +38,7 @@ const config = {
   },
   scripts: {
     patterns: [...npmPath, ...nodePath],
-    ignore: [
-      '**/node_modules/**',
-      // Built files and React repo clone
-      'scripts/bench/benchmarks/**',
-      // shims & flow treat as ESNext
-      'scripts/flow/*.js',
-      'scripts/rollup/shims/**/*.js',
-    ],
+    ignore: ['**/node_modules/**'],
```

Comment:
I think remove the scripts in ignore will cause other issues. Some patterns are covered (like `'scripts/flow/*.js'`) will run on both prettier process.

Because we run prettier twice (`default` -> `scripts`), the results seem work as our expect, but actually some changes on default prettier is covered by scripts prettier.

Maybe we can expand the scripts pattern to make it more clear?

```
const nodePath = [
  'scripts/babel/*.js',
  'scripts/circleci/*.js',
  'scripts/error-codes/*.js',
  ...
];
```

Reply:
I see. Let me look at that.

I think I can take this PR from here, thanks a lot!

Reply:
You're welcome. Thanks for your patience.